tools-reference/echo/text.xml

```diff
@@ -29,10 +29,10 @@ the same as the former, but they won't print the trailing newline
 
 <p>
 All usage of the form <c>echo ${somevar} | grep substring</c> just to
-check if the contents of the <c>${somevar}</c> variable
-contains <c>substring</c>, or more often, <c>echo ${somevar} |
+check if the content of the <c>${somevar}</c> variable
```

Comment:
The changes in this file feel more like style and should perhaps be split into their own commit.

Reply:
true, I got carried away :-)
I will keep the "third person S correction" in this commit and move the other stuff (commonly, rarely) to what else I'm preparing